assets/pages/contributions-thankyou/monthlyContributionsPending.jsx

```diff
@@ -0,0 +1,68 @@
+// @flow
```

Comment:
Pending is a version of the monthly contributions thank you page. The only distinction is slightly different copy.